src/github/branch.ts

```diff
@@ -29,7 +29,7 @@ export const deleteBranch = async (context, jiraClient): Promise<void> => {
 	}
 
 	await jiraClient.devinfo.branch.delete(
-		context.payload.repository.id,
+		context.payload.repository?.id,
```

Comment:
I don't think it will work if we don't have repo the branch wouldn't be deleted. (Jira probably return 404). Maybe we can just discard the evnet and log an error in such case?